lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,688 @@
+<div>
```

Comment:
@AlasdairSwan is there a way to do nested templates, or can we come up with an alternate architecture? This file is *too* large. We need a way to maintain a separation of concerns (e.g. basic details, credit, xseries, certificates, etc.)